packages/react-core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -18,6 +18,14 @@ export interface DropdownProps extends ToggleMenuBaseProps, React.HTMLProps<HTML
   isPlain?: boolean;
   /** Indicates where menu will be aligned horizontally */
   position?: DropdownPosition | 'right' | 'left';
+  /** Indicates how the menu will align at screen size breakpoints */
+  alignments?: {
```

Comment:
Is this missing `default`?

Reply:
`default` is effectively the `position` attribute. However `alignments` is set on a different element than `position`, so I left `default` out since I'm not sure it would work the same.

Reply:
I think dow the road if we have a breaking change we will want to combine the two props.  To me it seems a little confusing having the two.

Reply:
> However alignments is set on a different element than position

@kmcfaul do you mean the classes `alignments` add are on a different element than the classes added by `position`? If so, they should be the same. Looks like `position` adds classes to the `.pf-c-dropdown` element, but doesn't need to.